src/pages/components/theme-provider.mdx

```diff
@@ -0,0 +1,113 @@
+---
+title: Theme provider
+description: >
+  The Theme provider is a wrapper that provides theming values to all of the
+  components it contains.
+package: '@zendeskgarden/react-theming'
+components:
+  - theming/src/elements/ThemeProvider.tsx
+---
+
+<Usage>
+  <Use>
+    <ul>
+      <li>
+        <Markdown>
+          As the root component of the Garden DOM, where it provides global context for
+          [RTL](/components/theme-object#rtl) layout
+        </Markdown>
+      </li>
+      <li>As a nested wrapper for a subset of components that require themed styling</li>
+    </ul>
+  </Use>
+</Usage>
+
+## How to use it
+
+### Default
+
+The typical usage of a theme provided to styled components.
+
+import Default from '../../examples/theme-provider/ThemeProviderDefault.tsx';
+import DefaultCode from '!!raw-loader!../../examples/theme-provider/ThemeProviderDefault.tsx';
+
+<CodeExample code={DefaultCode}>
+  <Default />
+</CodeExample>
```

Comment:
The examples look good on the website but seem like it's rendered differently on CodeSandbox. Anyone else experiencing this?

<img width="1306" alt="Screen Shot 2020-11-03 at 12 43 51 PM" src="https://user-images.githubusercontent.com/1811365/98038703-6cc32c80-1dd2-11eb-9ae0-9aa393d95ee1.png">

Reply:
Ah, I think it's because I'm using `css` rather than `style`. I'll get this fixed.